frontend/package.json

```diff
@@ -20,8 +20,7 @@
     "@angular/platform-browser": "^4.0.0",
     "@angular/platform-browser-dynamic": "^4.0.0",
     "@angular/router": "^4.0.0",
-    "@ng-bootstrap/ng-bootstrap": "^1.0.0-beta.1",
-    "bootstrap": "^4.0.0-beta",
```

Comment:
@radoslawroszkowiak Do you remember why you removed this package? It doesn't build without it.

Reply:
I do. We were checking if these styles do not colllide with anything ng-bootstrap provides as the result bootstrap styles on the page have been incomplete. I don't think we wanted to commit it, after all, as it turned out not to be an issue.

Regardless, I think removing bootstrap npm package from the bundle and using bootstrap CDN is a better idea.

Reply:
Oh, I've just noticed - I meant just bootstrap package. I think this ng-bootstrap was removed accidently here?